js/src/components/RectangleGraphNode.vue

```diff
@@ -180,5 +180,11 @@ export default class RectangleGraphNode extends Vue {
     this.truncatedText = this.text;
     this.fitText();
   }
+
+  @Watch("simplifyGraph")
```

Comment:
I think a comment explaining what simplify graph does would be good. "Simplify graph" doesn't really describe what happens to the graph.